airbyte-integrations/connectors/source-sendgrid/sample_files/configured_catalog.json

```diff
@@ -291,10 +291,12 @@
             }
           }
         },
```

Comment:
we don't need to have two configured catalogs, lets leave only one in integration_tests

Reply:
Integration tests fail if not sample_files/configured_catalog.json exists
```
    def _opener(self, name, flags, mode=0o666):
        # A stub for the opener argument to built-in open()
>       return self._accessor.open(self, flags, mode)
E       FileNotFoundError: [Errno 2] No such file or directory: '/test_input/sample_files/configured_catalog.json'
```
May I put it back, or perhaps it better to remove it from `integrations_tests` instead?